### ISSUE TITLE
'helm list' doesn't seem to filter by user

### ISSUE THREAD

**Issue by gtaylor** [app-registry/helm-plugin#19]
```
$ helm registry list -u ant31 quay.io
<full list of repos instead of just ant31's>
```

**Comment by ant31** [comment]
@gtaylor you can use the -o; 
```
helm registry list -o ant31 quay.io
```

but from you suggestion we will replace that on the next release by
```
helm registry list quay.io/ant31 
```

**Comment by ant31** [comment]
the `-u` isn't implemented on quay.io. 

It should show all repositories a user have explicitly access to (at least read permission) include privates if the request-user have perm too.

**Comment by venezia** [comment]
It doesn't seem to list _all_ repositories that I may have access to.  I created a few public application repositories but they do not appear in the list.

`helm registry list quay.io|grep venezia`

returns nothing

**Comment by ant31** [comment]
@venezia are you sure they are public. By default it's private and you need to set it to public. 

`helm registry list quay.io`   without option returns only the 'first' page, it the future it will probably be ordered by 'highlight' or number of downloads. 

You can use: 
helm registry list quay.io --search operator
(it throws a 500 if there is no result, it's going to be fixed this week)

Private repos are never returned yet.

**Comment by venezia** [comment]
You're right - it is a private repo issue.  Thanks.  Will private repos eventually be surfaced?

**Comment by ryanwalls** [comment]
@ant31 How do I see private repos?  Related... how would I make a repo public?

**Comment by ant31** [comment]
@ryanwalls 
1. You can't 'list' private repos yet
2. make a repo public:  the command is missing

By default apps are private on quay.io, to switch you can go on quay.io in the setting tab of your application

**Comment by ryanwalls** [comment]
@ant31 Thanks for the clarification.  To further clarify, there is no way to see what private repos you have, even from the quay.io UI?

**Comment by ant31** [comment]
@ryanwalls I was speaking from the CLI, also those features (in the cli) will be added

On quay.io you can see all your private/public repo:  on the page https://quay.io/application/
